std/parallelism.d

```diff
@@ -2803,6 +2812,120 @@ public:
         }
     }
 
+    /** Implements the homonym function (also known as $(D accumulate), $(D
+        compress), $(D inject), or $(D foldl)) present in various programming
+        languages of functional flavor.
+
+        This is functionally equivalent to $(LREF reduce) except the range
+        parameter comes first and there is no need to use $(REF_ALTTEXT
+        $(D tuple),tuple,std,typecons) for multiple seeds.
+
+        Params:
+            functions = One or more functions
+
+        Returns:
+            The accumulated result as a single value for single function and as a tuple of values for multiple functions
+
+        See_Also:
+            $(LREF reduce)
```

Comment:
std.algorithm.iteration.fold?

Reply:
I really meant TaskPool.reduce here because std.algorithm.iteration.fold does not have the peculiarities of this module (mostly about the seeds).

Reply:
Oh I meant: how about mentioning std.algorithm.iteration.fold too? (sorry for the brevity, I'm on my phone).
Btw I think it would be better to use `TaskPool.reduce` as LREF here, but from what I can see there's only `MYREF`, but no `LREF_ALTTEXT` yet :/

- https://github.com/dlang/dlang.org/blob/master/std.ddoc

Also at least for Ddoc the link would be `#.TaskPool.reduce`, so maybe the ugly `MYREF` works already?